realm/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -173,10 +173,10 @@ public Boolean call() throws Exception {
         while (!isReady.get()) {
             Thread.sleep(5);
         }
-        Thread.sleep(100); 
+        Thread.sleep(100);
 
         // Trigger OnRealmChanged on background thread
-        Realm realm = Realm.getInstance(getContext());
+        realm = Realm.getInstance(getContext());
```

Comment:
Because these tests are so specialised that doing that would decrease readability. Only reason they are class variables in the first place is to make sure the Realm gets closed in case of a test failure which can impact other tests if the Realm is not closed.